test/testsizeof.cpp

```diff
@@ -735,7 +735,7 @@ class TestSizeof : public TestFixture {
         check("void f() {\n"
               "  int size = sizeof(void);\n"
               "}");
-        ASSERT_EQUALS("[test.cpp:2]: (portability) Behaviour of 'sizeof(void)' is not covered by the ISO C standard.\n", errout.str());
+        ASSERT_EQUALS("[test.cpp:2]: (portability) '*)' is of type 'void', the behaviour of 'sizeof(void)' is not covered by the ISO C standard.\n", errout.str());
```

Comment:
all other test cases seem good to me.. I just wonder what happened here.

Reply:
I updated the expected text output here now.

Reply:
do you know why we get such strange warning. I have not debugged it but I don't spontanously see where `*)` comes from.

Reply:
I think that in CheckSizeof::sizeofVoid (line 407) we could check not only for  Token::simpleMatch(tok, "sizeof ( )" but also Token::simpleMatch(tok, "sizeof (void)". Then it will report issue as sizeofVoid error instead of sizeofDereferencedVoidPointerError.

Reply:
I have pushed a fix for this now. Verified it locally.